### PR TITLE
BDHQIT-1570: infrabox change for single job restart

### PR DESCRIPTION
**Pull request by BigNuoLi**
Restart a single Infrabox job without rerun its children.

### CODE REVIEW

**Comment by BigNuoLi** [comment]
Merge the code changes from Berry's PR - increase project secret count limit to 200 #410

**Comment by BigNuoLi** [comment]
fix the issue on test case - test_restart_job

**Comment by BigNuoLi** [comment]
Please help to merge.

**Comment by BigNuoLi** [comment]
BDHQIT-1674: add new infrabox function for rerun a single job

**Comment by BigNuoLi** [comment]
Add a new function "Rerun" to rerun a job without restarting its children jobs.

API:  https://infrabox.datahub.only.sap/api/v1/projects/{PROJECT_ID}/jobs/{INFRABOX_JOB_ID}/rerun

File: src/api/handlers/projects/jobs.py [hunk 1]
```diff
@@ -383,6 +383,220 @@ def get(self, project_id, job_id):
...
+        single_job += g.db.execute_many_dict('''
+            SELECT id, build_id, type, dockerfile, name, project_id, dependencies,
+                    repo, env_var, env_var_ref, build_arg, deployment, definition, cluster_name
+            FROM job
+            WHERE id = %s;
+        ''', [job_id])
+
+        old_id_job = {}
+        loop_jobs = single_job
+        for j in loop_jobs:
```
Comment (iberryful) [inline src/api/handlers/projects/jobs.py:530]: `loop_jobs` is not needed any more since we won't restart children jobs.

Comment (BigNuoLi) [reply src/api/handlers/projects/jobs.py:530]: thanks Berry, I will remove this useless variable

**Pull request merged**